pkg/elemental/components/BuildMedia.vue

```diff
@@ -163,57 +193,70 @@ export default {
 <template>
   <div>
     <h3 class="build-iso-title">
-      {{ t('elemental.machineRegistration.edit.buildIsoTitle') }}
+      {{ t('elemental.machineRegistration.edit.buildMediaTitle') }}
     </h3>
     <div class="row mb-10">
       <div
         v-if="displayRegEndpoints"
-        class="col span-3"
+        class="col span-2"
       >
         <LabeledSelect
           v-model="registrationEndpointSelected"
           class="mr-20"
-          data-testid="select-registration-endpoint-build-iso"
+          data-testid="select-registration-endpoint-build-media"
           :label="t('elemental.machineRegistration.create.machineReg')"
           :placeholder="t('elemental.dashboard.regEndpointPlaceholder')"
           :options="registrationEndpointsOptions"
         />
       </div>
+      <div class="col span-2">
+        <LabeledSelect
+          v-model="buildMediaTypeSelected"
+          class="mr-20"
+          data-testid="select-media-type-build-media"
+          :label="t('elemental.machineRegistration.edit.mediaType')"
+          :placeholder="t('elemental.machineRegistration.edit.mediaTypePlaceholder')"
+          :options="buildMediaTypes"
+          option-key="value"
+        />
+      </div>
```

Comment:
When viewing this component from the dashboard view, this input should be disabled when a registration endpoint is not selected as the buildMediaOsVersions will always be empty until a new buildMediaTypeSelected is selected.

Reply:
We don't need to do that because the `build media` button is disabled unless the 3 elements are selected (in the dashboard view case - check `isBuildMediaBtnEnabled`).

Also, it doesn't have any implication with the registration endpoint select so both can be "available" for selection.